.dockerignore

```diff
@@ -1,8 +1,3 @@
-**/.git
```

Comment:
Multi stage build is used to build Dockerfile so we need not these rules anymore. These rules trigger `.git` directory is not properly copied inside a container.